code/modules/mob/living/simple_animal/abnormality/aleph/nobody_is.dm

```diff
@@ -264,24 +272,108 @@
 		release_damage = clamp(release_damage + damage, 0, release_threshold)
 		if(release_damage >= release_threshold)
 			ReleaseGrab()
-	last_heal_time = world.time + 10 SECONDS // Heal delayed when taking damage; Doubled because it was a little too quick.
+	if(!oberon_mode)
+		last_heal_time = world.time + 10 SECONDS // Heal delayed when taking damage; Doubled because it was a little too quick.
+	else
+		last_heal_time = world.time + 40 SECONDS // Heal delayed even more when taking damage; It was basically unkillable when it was 10 seconds.
 
 /mob/living/simple_animal/hostile/abnormality/nobody_is/Life()
 	. = ..()
 	if(!.)
 		return
-	if((last_heal_time + 1 SECONDS) < world.time) // One Second between heals guaranteed
-		var/heal_amount = ((world.time - last_heal_time)/10)*heal_percent_per_second*maxHealth
-		if(health <= maxHealth*0.3)
-			heal_amount *= 2
-		adjustBruteLoss(-heal_amount)
-		last_heal_time = world.time
+	if(!oberon_mode)
+		if((last_heal_time + 1 SECONDS) < world.time) // One Second between heals guaranteed
+			var/heal_amount = ((world.time - last_heal_time)/10)*heal_percent_per_second*maxHealth
+			if(oberon_mode)
+				if(abno_host && abno_host.currentlaw == "armor")//prevents regen from happening when you can't smack it with the type it hurts it the most. Range and melee laws aren't affected due to with titania you have soulmate.
+					last_heal_time = world.time + 40 SECONDS
+			if(health <= maxHealth*0.3)
+				heal_amount *= 2
+			adjustBruteLoss(-heal_amount)
+			last_heal_time = world.time
+
 	if(next_transform && (world.time > next_transform))
 		next_stage()
 	if(current_stage == 2) // Egg
 		var/obj/effect/temp_visual/decoy/D = new /obj/effect/temp_visual/decoy(get_turf(src), src)
 		animate(D, alpha = 0, transform = matrix()*1.2, time = 7)
+	if(disguise)
```

Comment:
Check if shelled is true as opposed to if disguise is true, as disguise can turn into a null value under certain conditions